awx/main/models/ha.py

```diff
@@ -159,9 +156,8 @@ def capacity(self):
     '''
     RelatedJobsMixin
     '''
-    def _get_active_jobs(self):
-        return UnifiedJob.objects.filter(instance_group=self,
-                                         status__in=ACTIVE_STATES)
+    def _get_related_jobs(self):
+        return UnifiedJob.objects.filter(instance_group=self)
```

Comment:
Nice removing all of the extra filtering since the mixin already applies this filter.